### DISCUSSION TITLE
Restore windows after restart Explorer?

### DISCUSSION THREAD

**Discussion by zaadstra** [valinet/ExplorerPatcher#2262] [Ideas]
Well, after the daily updates lately (thank you, thank you!), the wish has grown that after Explorer is restarted, and all open windows are closed.  Can the location be stored beforehand and then after patching the windows be reopened? Would be great :-)
Normally, I have a lot of windows opened, I have a very active desktop.

I am using the very ancient program ShellFolderFix for this now, but it mistakes by restoring the 4K screen positions to HD screen values, i.e. all windows are small and positioned in the left upper corner of the screen :-/
Other suggestions for this external tool are welcome as well.

**Comment by Imold** [comment]
That's not really within the scope of this project. But maybe one of these can help you 
[https://superuser.com/questions/1769490/restore-previous-folder-windows-after-restarting-explorer-exe](https://superuser.com/questions/1769490/restore-previous-folder-windows-after-restarting-explorer-exe) 
[https://superuser.com/questions/1700083/how-to-restart-explorer-without-losing-open-windows](https://superuser.com/questions/1700083/how-to-restart-explorer-without-losing-open-windows)

**Comment by zaadstra** [comment]
Needed it today for an update of EP.  The second link works like a charm for me.  

I created two batches and only removed /min twice in the restore part.
